Sample-WPF/Controls/FaceDetectionPage.xaml.cs

```diff
@@ -203,69 +204,123 @@ public string SelectedFile
             dlg.DefaultExt = ".jpg";
             dlg.Filter = "Image files (*.jpg, *.png, *.bmp, *.gif) | *.jpg; *.png; *.bmp; *.gif";
             var result = dlg.ShowDialog();
-
+            
             if (result.HasValue && result.Value)
             {
                 // User picked one image
-                var imageInfo = UIHelper.GetImageInfoForRendering(dlg.FileName);
-                SelectedFile = dlg.FileName;
-
+                var pickedImagePath = dlg.FileName;
+                var renderingImage = UIHelper.LoadImageAppliedOrientation(pickedImagePath);
+                var imageInfo = UIHelper.GetImageInfoForRendering(renderingImage);
+                SelectedFile = renderingImage;
+                
                 // Clear last detection result
                 ResultCollection.Clear();
                 DetectedFaces.Clear();
                 DetectedResultsInText = string.Format("Detecting...");
 
-                MainWindow.Log("Request: Detecting {0}", SelectedFile);
+                MainWindow.Log("Request: Detecting {0}", pickedImagePath);
                 var sw = Stopwatch.StartNew();
 
                 // Call detection REST API
-                using (var fileStream = File.OpenRead(SelectedFile))
-                {
-                    try
-                    {
-                        MainWindow mainWindow = Window.GetWindow(this) as MainWindow;
```

Comment:
Please have an extra check why there are so many `unchanged` changes.
Just to confirm no unexpected changes happen.